includes/callbacks.inc

```diff
@@ -90,20 +91,21 @@ function islandora_entities_autocomplete($type) {
     'fl' => array($mappings[$type]['title'], 'PID'),
```

Comment:
Fair point; I'm ignoring the overall weirdness here carrying over from previous functionality so we might consider some old fashioned module cleanup to be part of a new ticket